packages/addons/addon-depends/inadyn/libconfuse/package.mk

```diff
@@ -2,11 +2,11 @@
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="libconfuse"
-PKG_VERSION="3.2.2"
-PKG_SHA256="2cf7e032980aff8f488efba61510dc3fb95e9a4b9183f985dea457a5651b0e2c"
-PKG_LICENSE="https://github.com/martinh/libconfuse/blob/master/LICENSE"
-PKG_SITE="https://github.com/martinh/libconfuse"
-PKG_URL="https://github.com/martinh/libconfuse/archive/v$PKG_VERSION.tar.gz"
+PKG_VERSION="3.3"
+PKG_SHA256="cb90c06f2dbec971792af576d5b9a382fb3c4ca2b1deea55ea262b403f4e641e"
+PKG_LICENSE="https://github.com/libconfuse/libconfuse/blob/master/LICENSE"
```

Comment:
```suggestion
PKG_LICENSE="ISC"
```

Reply:
needs https://github.com/spdx/license-list-data/blob/master/text/ISC.txt copied to `LE/licenses/`

Reply:
License updated

Reply:
Note to self: no rushing to fix PR before heading the the office half 😴. Fixed.